### PR TITLE
W-10993540 rate limiting

### PR DESCRIPTION
**Pull request by hannanelson**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/mq-apis.adoc [hunk 1]
```diff
@@ -354,6 +354,9 @@ This information is not real-time and has a latency.
 
 `/environments/ENVIRONMENT_ID?startDate=...&endDate=...&period=...`
 
+[NOTE]
+Anypoint MQ limits retrieving stats by environment to 10 requests per organization, per minute.
```
Comment (GouthamiKondakindi) [inline modules/ROOT/pages/mq-apis.adoc]: Same comment as the release notes PR..

We use the term "transactions per minute" in MQ Admin docs: https://docs.mulesoft.com/mq/mq-apis#mqadminapi
> If you call the Anypoint MQ Admin API for distinct environments within one minute, the API throttles your requests to 50 transactions per minute (TPM).

Should we use the same term here for consistency?

Comment (hannanelson) [reply modules/ROOT/pages/mq-apis.adoc]: Yep

**Review by GouthamiKondakindi — DISMISSED** [review]
LGTM

**Comment by hannanelson** [comment]
See #116

**Pull request closed without merging**